Emby.Server.Implementations/Library/Resolvers/PhotoResolver.cs

```diff
@@ -25,7 +23,7 @@ public class PhotoResolver : ItemResolver<Photo>
         private readonly NamingOptions _namingOptions;
         private readonly IDirectoryService _directoryService;
 
-        private static readonly HashSet<string> _ignoreFiles = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+        private static readonly string[] _ignoreFiles = new[]
```

Comment:
Why change the type? Is Array.Contains faster than HashSet?

Reply:
For very small arrays (< 5 elements) it can be but I chnged it here bc we don't even call `.Contains` anymore